ZcashFoundation/zcash-grant-system
# Email admin + more email templates

Some infrastructure and work towards #12.

### What This Does

* Adds an admin page to preview emails
* Adds email templates for proposal accepted / rejected, and contribution confirmed
* Fixes some issues with existing templates

### Steps to Test

1. Run the admin on the `email-admin` branch
2. Go to the emails tab, confirm all emails show up
3. Click on each email, confirm the email renders and that there are no issues with the templates
4. Review the wiki instructions for making a new email: https://github.com/dternyak/zcash-grant-system/wiki/Creating-an-Email

## Screenshots

### Email admin

<img width="1277" alt="screen shot 2019-01-06 at 3 43 10 pm" src="https://user-images.githubusercontent.com/649992/50741330-d750b480-11c9-11e9-8b94-681ae269dee6.png">

### Example email (contribution confirmed)

![fireshot capture 21 - admin - grant io - http___localhost_3500_emails_contribution_confirmed](https://user-images.githubusercontent.com/649992/50741335-e8012a80-11c9-11e9-8b59-c5de6d73c573.png)
